vyper/venom/venom_to_assembly.py

```diff
@@ -301,7 +385,7 @@ def _generate_evm_for_instruction(
                 stack.poke(0, ret)
             else:
                 stack.poke(depth, ret)
-            return assembly
+            return apply_line_numbers(inst, assembly)
```

Comment:
i feel like apply_line_numbers could just be applied globally in the main venom_to_assembly routine